zerver/openapi/openapi.py

```diff
@@ -218,6 +218,19 @@ def get_openapi_summary(endpoint: str, method: str) -> str:
     return openapi_spec.openapi()["paths"][endpoint][method.lower()]["summary"]
 
 
+def get_endpoint_from_operationid(operationid: str) -> str:
+    print(operationid)
+    for each_endpoint in openapi_spec.openapi()["paths"]:
+        for each_method in openapi_spec.openapi()["paths"][each_endpoint]:
+            if (
+                "operationId" in openapi_spec.openapi()["paths"][each_endpoint][each_method]
+                and openapi_spec.openapi()["paths"][each_endpoint][each_method]["operationId"]
+                == operationid
+            ):
+                return each_endpoint + ":" + each_method
+    return ""
```

Comment:
I'd recommend returning a tuple, rather than a `:`-delimited string that the caller needs to parse.

Should we throw an AssertionError if we didn't find it?

Reply:
Changed it to tuple. Also, AssertionError makes sense in such case -- added that